src/oidcc_provider_configuration_worker.erl

```diff
@@ -227,10 +228,9 @@ handle_continue(
 handle_info(backoff_retry, State) ->
     {noreply, State, {continue, load_configuration}};
 handle_info(configuration_expired, State) ->
-    {noreply, State#state{configuration_refresh_timer = undefined, jwks_refresh_timer = undefined},
-        {continue, load_configuration}};
+    {noreply, State, {continue, load_configuration}};
```

Comment:
At this point the configuration timer expired, why keep a reference to the config timer?

Reply:
You're right, amended.